src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/ConvertToLibraryImportAnalyzerTests.cs

```diff
@@ -64,6 +61,20 @@ public async Task TypeRequiresMarshalling_ReportsDiagnostic(Type type)
                     .WithArguments("Method_Return"));
         }
 
+        [Fact]
+        public async Task FunctionPointer_ReportsDiagnostic()
+        {
+            string source = DllImportWithType("delegate* unmanaged<void>");
```

Comment:
Confirming that a function pointer degrades to a `void*`, right?

Reply:
Yep.